diffpy/pdfmorph/tests/test_pdfmorphapp_exceptions.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+
+import unittest
+import pytest
+from pathlib import Path
+
+
+from diffpy.pdfmorph.pdfmorphapp import create_option_parser, single_morph, multiple_morphs
+
+thisfile = locals().get('__file__', 'file.py')
+tests_dir = Path(thisfile).parent.resolve()
+testdata_dir = tests_dir.joinpath("testdata")
+testsequence_dir = testdata_dir.joinpath("testsequence")
+nickel_PDF = testdata_dir.joinpath("nickel_ss0.01.cgr")
+serial_JSON = testdata_dir.joinpath("testsequence_serialfile.json")
+
+class TestExceptions(unittest.TestCase):
```

Comment:
Will change to pytest; additionally, the three files (1) `test_pdfmorphapp_exceptions.py`, (2) `test_morphsequence.py` and (3) `test_parser.py` will be moved to a new file `test_pdfmorphapp.py` to follow convention. This will be done either in the change_libraries branch or another commit.